I have seen that CI behaviour. (https://github.com/home-assistant/core/pull/41351, https://github.com/home-assistant/core/pull/41257) 
The test seems to be related to the home kit integration. 
```
=========================== short test summary info ============================
FAILED tests/components/homekit/test_homekit.py::test_reload[pyloop] - Failed...
Error: Process completed with exit code 1.
```

--- a/homeassistant/components/plaato/strings.json
+++ b/homeassistant/components/plaato/strings.json
@@ -8,7 +8,7 @@
     },
     "abort": {
       "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
-      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+      "not_internet_accessible": "Your Home Assistant instance needs to be accessible from the internet to receive messages from Plaato Airlock."
     },
     "create_entry": {
       "default": "To send events to Home Assistant, you will need to setup the webhook feature in Plaato Airlock.\n\nFill in the following info:\n\n- URL: `{webhook_url}`\n- Method: POST\n\nSee [the documentation]({docs_url}) for further details."

